index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav id="navbar">
+        <ul>
+          <li>
+            <a href="">About</a>
+          </li>
+          <li>
+            <a href="#projects">Work</a>
+          </li>
+          <li>
+            <a href="#contact">contact</a>
+          </li>
+        </ul>
+      </nav>
+      <section  id="welcome-section">
+        <h1>Hey I am Roshan<br>
+          a web developer
+        </h1>
+      </section>
+      <section id="projects">
+        <div class="project-tile">
+          <a href= "#">These are some of my projects</a>
+        </div>
+        <div class="photo">
+          <img src="1.jpg" alt="image">
+          <img src="2.png" alt="image">
+          <img src="3.png" alt="image">
```

Comment:
- [ ]  Change the alternative name for the image.

Reply:
Please try to make use of a better way to communicate about required changes while giving reviews. Could you please rephrase the sentence to have a form of a suggestion? Try to always explain why a given change is required because you should.